klippa/src/vorg.rs

```diff
@@ -0,0 +1,108 @@
+//! impl subset() for VORG
+use crate::serialize::Serializer;
+use crate::{Plan, Subset, SubsetError};
+use write_fonts::{
+    read::{tables::vorg::Vorg, FontRef, TopLevelTable},
+    types::GlyphId,
+    FontBuilder,
+};
+
+// reference: subset() for VORG in harfbuzz
+// <https://github.com/harfbuzz/harfbuzz/blob/22fbc7568828b9acfd116be44b2d77d56d2d448b/src/hb-ot-vorg-table.hh#L90>
+impl Subset for Vorg<'_> {
+    fn subset(
+        &self,
+        plan: &Plan,
+        _font: &FontRef,
+        s: &mut Serializer,
+        _builder: &mut FontBuilder,
+    ) -> Result<(), SubsetError> {
+        s.embed(self.version())
+            .map_err(|_| SubsetError::SubsetTableError(Vorg::TAG))?;
```

Comment:
The `map_err` duplication is a bit noisy. If we’re just converting all errors to the same thing, it might be cleaner to move the logic to a private function in this module that returns `Result<(), SerializeErrorFlags>` and call that from the public API, doing the error conversion once.